lib/integrations/Daily/DailyVideoApiAdapter.ts

```diff
@@ -97,6 +97,23 @@ const DailyVideoApiAdapter = (credential: Credential): VideoApiAdapter => {
     // added a 1 hour buffer for room expiration and room entry
     const exp = Math.round(new Date(event.endTime).getTime() / 1000) + 60 * 60;
     const nbf = Math.round(new Date(event.startTime).getTime() / 1000) - 60 * 60;
+    const scalePlan = process.env.DAILY_SCALE_PLAN;
```

Comment:
this looks like it's not checking for "enable"

we should just check for boolean instead (true)

Reply:
I updated it to a true